package.json

```diff
@@ -24,10 +24,13 @@
     "preversion": "npm run test && npm run check-changelog && npm run check-only-changelog-changed",
     "tag": "git tag v$npm_package_version"
   },
-  "dependencies": {
-    "acorn": "^5.0.0"
+  "peerDependencies": {
+    "acorn": "^6.0.0",
+    "acorn-walk": "^6.0.0"
```

Comment:
It feels weird to have this be a peer depencency, but have this functionality be optional. Can this either be a regular dependency, or can we just make a note in the readme that if you're going to use the walk plugin you need to install this?

Reply:
Would dropping the re-export from `walk.js` (along with the peer dep) be acceptable to you? That way, it's a little more obvious that you'll need to install acorn-walk yourself.

Reply:
As proposed in 9841d70

Reply:
Yeah I think that's reasonable.